buildchain/buildchain/targets/package.py

```diff
@@ -106,12 +106,13 @@ def srcdir(self) -> Path:
     @property
     def spec(self) -> Path:
         """.spec file path."""
-        return constants.ROOT/'packages'/'{}.spec'.format(self.name)
+        return constants.ROOT/'packages'/'redhat'/'{}.spec'.format(self.name)
 
     @property
     def meta(self) -> Path:
         """.meta file path."""
-        return self.rootdir/'{}.meta'.format(self.name)
+        #return self.rootdir/'redhat'/'{}.meta'.format(self.name)
```

Comment:
the comment can be removed I guess

Reply:
This is wrong, as we have seen.

Instead of fixing it in a later commit, I would squash the commit `Change repositories target ` into this one.